dnf-behave-tests/dnf/plugin-enable.feature

```diff
@@ -1,42 +1,56 @@
+@dnf5
+#TODO(amatej): replace log checking with https://github.com/rpm-software-management/dnf5/issues/1335
 Feature: Plugin enablement by config file and command line option
 
 
 Background:
 Given I do not disable plugins
 
 
-Scenario: Verify that debuginfo-install plugin is enabled
- When I execute dnf with args "-v repolist"
- Then stdout contains "Loaded plugins:.* debuginfo-install.*"
+Scenario: Verify that actions plugin is enabled
+ When I execute dnf with args "repo list"
+ Then file "/var/log/dnf5.log" contains lines
+      """
+      .* Loaded libdnf plugin "actions" .*
+      """
 
 
 Scenario: Disable enabled debuginfo-install plugin from command line
```

Comment:
There is a forgotten leftover of `debuginfo-install`.

Reply:
Thanks! Fixed.